chef/cookbooks/apache2/definitions/apache_site.rb

```diff
@@ -19,7 +19,14 @@
 
 define :apache_site, enable: true do
   include_recipe "apache2"
-  if node.platform != "suse"
+  if node[:platform_family] == "suse"
+    if not params[:enable] then
```

Comment:
Style/MultilineIfThen: Do not use `then` for multi-line `if`. (https://github.com/bbatsov/ruby-style-guide#no-then)